cmd/casload/BUILD

```diff
@@ -1,6 +1,16 @@
-go_package()
+go_package(name="src")
 
 go_binary(
     name="bin",
     output_path="casload",
 )
+
+docker_image(
+    name="casload",
```

Comment:
Maybe `go_package` should remain with the default target name for this directory? And then this `docker_image` target would be something like "docker" or "image."

Reply:
that was what I initially did, but that name is used when publishing to docker hub, so I wanted to keep that, see: https://github.com/toolchainlabs/remote-api-tools/pull/71

Reply:
Why not use the `repository` field to set the published name?

Reply:
good point, will do.